scripts/listapranzo.js

```diff
@@ -85,6 +85,29 @@ module.exports = function (robot) {
     order.users[user.id] = dishes;
   };
 
+  var fuzzyMatch = function (dish, menuline) {
+    dish = dish.trim().toLowerCase().replace(" ", "");
+    var key = "";
+    for (var i = 0; i < dish.length; i++)
+      key = key + dish[i] + ".*"
```

Comment:
@rasky Non è il contrario? Cioè che se una entry è `pqqqqqqqqqqenne` allora `penne` lo matcha? A me questo comportamento piace, non ho voglia di contare quante `q` devo premere. Vero che con solo substring `qenne` funzionerebbe, ma fuzzy è più generico e comodo secondo me - che è il motivo per cui è usato in praticamente tutti i file finder ever.